buddyboss/buddyboss-platform
# PROD-744 Fix 2903 Issue with youtube preview whenever link has space at the end

Fixed: preview becomes different compared to normal URL embed of YouTube link.

### All Submissions:




Fixes #2903.

### How to test the changes in this Pull Request:

1. Paste the YouTube share link and
2. Remove old preview, if already
3. Type space at end of the YouTube share link
4. Preview should appear earlier it was not working due to &nbsp; at the end of the link

### Proof Screenshots or Video

http://somup.com/crQuiBrPjY

### Other information:




### Changelog entry

> Removed the &nbsp; from the URL. Preview should appear now earlier it was not working due to &nbsp; at the end of the link
